docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -122,11 +127,11 @@ To generate an API Key using the Cloud UI:
 
 :::note
 
-Temporal supports up to a 90-day duration for API Keys.
+Temporal currently supports up to a 90-day duration for API Keys. The expiration will be extended in a future release.
```

Comment:
Do we need to include this? Fine with either way though. 
"The expiration will be extended in a future release."

Reply:
IMO it's best to never mention future changes in docs. We may do it. We may not. Make no promises. Similarly avoid "currently". The docs show what is accurate to this feature.

Same question/ticketing about capitalization for API Keys. 

Suggested rewording:

Temporal supports API keys lifespans up to 90 days. Your key will be valid during this period unless it is revoked or the account is deactivated.